command/vsm_stats.go

```diff
@@ -427,17 +428,17 @@ func StatsOutput(c *VsmStatsCommand, annotations *Annotations, args []string, st
 
 	} else {
 
-		// Printing in tabular form
-		//	fmt.Printf("%+v\n\n", annotation)
-		data, err := json.MarshalIndent(annotation, "", "\t")
-
+		// Printing using template
+		tmpl, err := template.New("test").Parse("IQN     : {{.IQN}}\nVolume  : {{.Volume}}\nPortal  : {{.Portal}}\nSize    : {{.Size}}")
+		if err != nil {
+			panic(err)
+		}
+		err = tmpl.Execute(os.Stdout, annotation)
 		if err != nil {
-
 			panic(err)
 		}
 
-		os.Stdout.Write(data)
-
+		// Printing in tabular form
 		q := tabwriter.NewWriter(os.Stdout, minwidth, maxwidth, padding, ' ', tabwriter.AlignRight|tabwriter.Debug)
 
 		fmt.Fprintf(q, "\n\nReplica\tStatus\tDataUpdateIndex\t\n")
```

Comment:
This is required to print the output in tabular format as suggested by @prateekpandey14.